protocol/clproto_cpp/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ project(clproto VERSION 4.0.0)
 
 set(CMAKE_CXX_STANDARD 17)
 
+include(FindProtobuf)
```

Comment:
Where is that coming from?

Reply:
https://stackoverflow.com/questions/10010398/how-to-link-google-protobuf-libraries-via-cmake-on-linux

I had linker errors yesterday on the franka PC and this resolves the library path correctly

Reply:
Thanks for the link. First time I see this type of cmake command. I am surprise that the `find_package(Protobuf REQUIRED)`  is not sufficient for this.